dipy/denoise/tests/test_ascm.py

```diff
@@ -124,6 +124,6 @@ def test_ascm_accuracy():
         rician=True)
 
     S0n = np.array(adaptive_soft_matching(test_data,
-                                          den_small, den_large, sigma[0]))
+                                          den_small, den_large, sigma))
 
-    assert_array_almost_equal(S0n, test_ascm_data_ref)
+    assert_array_almost_equal(S0n, test_ascm_data_ref, decimal=4)
```

Comment:
The other solution to fix https://github.com/dipy/dipy/actions/runs/6655794234/job/18086859408?pr=2941#step:9:3896 is to set
```
sigma = np.round(estimate_sigma(test_data, N=4).item(), decimals=5).astype(np.float32)
```

But I think the proposed one is less convoluted.